main.go

```diff
@@ -28,6 +29,9 @@ func main() {
 		context.Rest.SetRoutes(c.Routes())
 	}
 
+	// Store self information in the store
+	peer.AddSelfDetails()
```

Comment:
Self always indicates the node and volume is something which is shared and not owned by a single node in gluster terminology. I can't buy in this comment.